packages/gatsby-theme-docs/src/components/code-example.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CodeBlock, ContentNotifications } from '@commercetools-docs/ui-kit';
+import useCodeExamples from '../hooks/use-code-examples';
+
+function CodeExample(props) {
+  const codeExamples = useCodeExamples();
+
+  const codeExample = codeExamples.find(example => {
+    return example.absolutePath.includes(props.file);
```

Comment:
I find filtering using `includes()` relatively adventurous because many can match (even if you would only look at the end of the string. 

Why not require an exact match?  (see below in the plugin for my proposal concerning filtering the graphQL by path prefix)

Reply:
We can use `endsWith`

```suggestion
    return example.absolutePath.endsWith(props.file);
```